python/paddle/nn/functional/__init__.py

```diff
@@ -162,6 +163,7 @@
     'conv3d',
     'conv3d_transpose',
     'pairwise_distance',
+    'pdist',
```

Comment:
which path do we recommend users to use? If `paddle.pdist` is recommended, it cannot be added to this \_\_all\_\_ list. if `paddle.nn.functional.pdist`, it cannot be added to the \_\_all\_\_ list in `python/paddle/__init__.py`

Reply:
Tks for you review, I think I prefer `paddle.pdist` path, I removed this line then.